libs/filamat/CMakeLists.txt

```diff
@@ -56,11 +56,13 @@ target_link_libraries(${TARGET} shaders filabridge filaflat utils)
 # ==================================================================================================
 # Compiler flags
 # ==================================================================================================
-target_compile_options(${TARGET} PRIVATE
-# TODO: use hidden by default and expose what we need (we'll need to do the same in dependencies)
-#        -fvisibility=hidden
-        $<$<PLATFORM_ID:Linux>:-fPIC>
-)
+if (NOT WIN32)
```

Comment:
You can drop the test here since there's already a test inside the target_compile_options